programs/uxd/src/state/controller.rs

```diff
@@ -22,7 +23,9 @@ pub const CONTROLLER_SPACE: usize = 8
     + 8 // unused
     + (32 * MAX_REGISTERED_MERCURIAL_VAULT_DEPOSITORIES)
     + 1
-    + 375;
+    + (32 * MAX_REGISTERED_CREDIX_LP_DEPOSITORIES)
+    + 1
+    + 246;
```

Comment:
the math here missed the `profits_total_collected`

Reply:
Nice catch..